autupdate.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+RUBY_REPOS_FILE="`pwd`/repos/ruby"
+JS_REPOS_FILE="`pwd`/repos/javascript"
```

Comment:
There has to be a better way to do this.  Halp!?

Reply:
Errrr. Seems fine to me? Any ideas @cbeer?

Reply:
on Jenkins, it's probably `$WORKSPACE`; `dirname $0` might be the fallback.

Reply:
Hmm, will that cause problems if trying to run locally?  (Or maybe that can't shouldn't be a concern of ours any longer?)